allensdk/brain_observatory/behavior/data_objects/metadata/behavior_ophys_metadata.py

```diff
@@ -3,23 +3,44 @@
 from pynwb import NWBFile
```

Comment:
What changed in this file?

Reply:
I think it's when I rebased my PR onto the updated RC branch, there were merge conflicts that I needed to resolve which marked the file as being touched, and my script to run black on changed files formatted it.